src/interface/report/Results/PlayerInfoGear.tsx

```diff
@@ -34,7 +40,7 @@ const PlayerInfoGear = (props: Props) => {
               className={`item-slot-${gearSlot}`}
             >
               <ItemLink id={item.id} quality={quality} details={item} icon={false}>
-                <Icon className="gear-icon icon" icon={item.icon} />
+                <Icon className={'gear-icon icon ' + qualityClass} icon={item.icon} />
```

Comment:
use a template string for this:

```suggestion
                <Icon className={`gear-icon icon ${qualityClass}`} icon={item.icon} />
```

string concat via `+` is loaded with potential fun in JS :)

Reply:
OH! Excelent information. Will keep that in the back of my head for the future :)